test/cpp/ext/otel/otel_test_library.cc

```diff
@@ -39,6 +39,105 @@
 #include "test/cpp/end2end/test_service_impl.h"
 #include "test/cpp/util/byte_buffer_proto_helper.h"
 
+namespace {
+
+template <typename T>
+std::string ToString(T value) {
+  return std::to_string(value);
+}
+
+template <>
+std::string ToString(bool value) {
+  return value ? "true" : "false";
+}
+
+template <>
```

Comment:
I don't think you need "template<>" - this is a function, so no need to specialize the template.

Reply:
removed